src/testing/builderbot/simulator/test_builderbot.argos.in

```diff
@@ -35,7 +35,8 @@
         <builderbot_system implementation="default" />
         <wifi implementation="default" />
       </sensors>
-      <params script="@CMAKE_BINARY_DIR@/testing/builderbot/test_builderbot.lua" />
+      <params script="@CMAKE_BINARY_DIR@/testing/builderbot/test_builderbot.lua" 
+              search_random_range="0"/>
```

Comment:
I would think that if this is the baseline example, all parameters would be the defaults, no?